shell/src/main/java/tachyon/shell/command/RmrCommand.java

```diff
@@ -46,6 +47,8 @@ void runCommand(TachyonURI path) throws IOException {
       TachyonFile fd = mTfs.open(path);
       mTfs.delete(fd, options);
       System.out.println(path + " has been removed");
+    } catch (InvalidPathException e) {
+      System.out.println(path + " does not exist");
```

Comment:
Currently AbstractTachyonFileSystem.open() throws an InvalidPathException when the file does not exist. That's why we need to catch this exception here.